test/fixtures/appforms/fixture_forms.js

```diff
@@ -27,4 +27,6 @@ module.exports = nock('https://apps.feedhenry.com')
   .delete('/api/v2/appforms/forms/someformid', '*')
   .reply(200, envReplies.crud)
   .post('/api/v2/appforms/forms/someformid/clone', '*')
+  .reply(200, envReplies.crud)
+  .get('/api/v2/appforms/forms/export', '*')
```

Comment:
We can have an empty file for now and add sample data from the backend endpoint later.

My test on the import endpoint is also wrong in this part, I'll change it to a 204 response with no content